client/mass/groups.js

```diff
@@ -218,6 +218,17 @@ class MassGroups {
 				deleteCallback()
 				this.tip.hide()
 			})
+		if (this.state.termdbConfig.scatterplots)
+			for (const plot of this.state.termdbConfig.scatterplots) {
+				menuDiv
+					.append('div')
+					.attr('class', 'sja_menuoption sja_sharp_border')
+					.text(`Open ${plot.name}`)
```

Comment:
rather than filter the scatter plot to only those samples in the list, we want to overlay the custom samplelst term on the plot. Karishma is using such an example in the figure

Reply:
Yes I know, I was trying to keep the original plot configuration so I added the filter instead. If this is what is needed I can customize it.

Reply:
Now it should work as expected